lib/routes/developers/register.js

```diff
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const auth = require('../../auth');
+const db = require('../../db');
+
+/*jshint camelcase: false*/
+module.exports = {
+  auth: {
+    strategy: auth.AUTH_STRATEGY,
+    scope: [auth.SCOPE_CLIENT_MANAGEMENT]
+  },
+  handler: function developerRegistration(req, reply) {
+    var email = req.auth.credentials.email;
+
+    return db.developerFind(email)
+      .then(
+        function(developer) {
+          if (developer) {
+            return reply(developer).code(200);
```

Comment:
Trying to register a developer that already exists is a 200? I'd expect some 400 error, personally...